docs/sources/tutorial/getting-started.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Beyla quick start tutorial
-menuTitle: Quick start tutorial
+title: Getting started with Beyla
```

Comment:
Renamed "quick start" to "getting started" to not generate confusion wit the "quickstart" guides.